Automattic/woocommerce-payments
# fix: payment sections settings sorting

Fixes https://github.com/Automattic/woocommerce-payments/issues/1583

#### Changes proposed in this Pull Request









I noticed that with the recent addition of the new payment gateway, the list of payment gateways got a bit messed up.
Unfortunately, it is not sufficient to _just_ hide the gateways with CSS. By only hiding them with CSS, the elements are still in the DOM and mess up the sorting plugin.

**PLEASE NOTE**: with these changes, to enable SEPA you will also need to set the "grouped settings" flag in WCPay dev

#### Testing instructions





- (without checking out this branch) Go to the [payment settings page](http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout)
- Notice how the arrows are a bit messed up: you can see arrows where they should be disabled: https://d.pr/i/5QEaKS
- Notice how the backgrounds of the table can also be inconsistent when interacting with the arrows
- Notice how interacting with the arrows can cause some unexpected behavior: 
![2021-04-14 18 04 41](https://user-images.githubusercontent.com/273592/114791373-0dd3ed00-9d4c-11eb-9536-954e4b020dd5.gif)
- Now check out this branch. The behavior should no longer be there.
- Also enable the "grouped settings" flag in WCPay dev - the behavior should be consistent


-------------------
